ompi/mca/osc/ucx/osc_ucx.h

```diff
@@ -24,7 +24,7 @@
 #define OSC_UCX_VERBOSE MCA_COMMON_UCX_VERBOSE
 
 #define OMPI_OSC_UCX_POST_PEER_MAX 32
-#define OMPI_OSC_UCX_ATTACH_MAX    32
+#define OMPI_OSC_UCX_ATTACH_MAX    48
```

Comment:
Any chance we can make this dynamically controllable via an MCA parameter? Or get rid of the limit entirely?